templates/cli/lib/config.js.twig

```diff
@@ -7,15 +7,18 @@ const JSONbig = require("json-bigint")({ storeAsString: false });
 class Config {
     constructor(path) {
         this.path = path;
+        this.configFileAbsent = null;
         this.read();
     }
 
     read() {
         try {
             const file = fs.readFileSync(this.path).toString();
             this.data = JSONbig.parse(file);
+            this.configFileAbsent = false;
```

Comment:
I don't think we need to add this property because it's another property that would need to kept in sync with the state of whether there's a config file or not. It's probably best to infer the information based on the data. We could infer the appwrite.json isn't present if either data is empty or there is no project id set.